Gemfile

```
@@ -0,0 +1,56 @@
+source "https://rubygems.org"
+
+ruby "3.4.3"
```

RuboCop (1.75.5)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)